backend/tournesol/views/comparison.py

```diff
@@ -95,6 +95,7 @@ def post(self, request, *args, **kwargs):
         """
         Create a new comparison associated with the logged user, in a given
         poll.
+        Add the comparison count via this route/function, no need to apply this logic in updates sections etc.
```

Comment:
Is this comment a kind of a personal reminder?

As this docstring is part of the public API documentation, I suggest to move this reminder elsewhere. If not it will be displayed to the API consumers, and I think this could be confusing.

See: https://api.tournesol.app/docs/#/users/users_me_comparisons_create

Reply:
You're right, I was trying different approaches and steps to better my understanding of the backend flow, so I might have added some comments here and there and now am unaware of them since they were committed in super early commits. Apologies for that, they were just for my own reminder.